frontend/src/views/WheresMyGene/components/HeatMap/style.ts

```diff
@@ -61,8 +78,23 @@ export const XAxisWrapper = styled.div`
 
 export const ChartWrapper = styled.div`
   position: absolute;
-  padding-left: ${CHART_LEFT_PADDING}px;
+  padding-left: ${CHART_PADDING_PX}px;
+  padding-right: ${CHART_PADDING_PX}px;
   padding-top: 5px;
   left: ${Y_AXIS_CHART_WIDTH_PX}px;
   top: ${X_AXIS_CHART_HEIGHT_PX}px;
 `;
+
+/**
+ * This is needed so that the heatmap scrollbar isn't covered by the right sidebar
+ */
+function isFmgOpen({ isFmgOpen = false }: { isFmgOpen: boolean }) {
+  return `calc(
+    100vw - ${
+      FILTERS_PANEL_EXPANDED_WIDTH_PX +
+      CONTENT_WRAPPER_LEFT_RIGHT_PADDING_PX * 2 +
+      (isFmgOpen ? CELL_INFO_SIDEBAR_WIDTH_PX : 0) -
+      25 // This number gets the scrollbar to the right edge of the viewport
```

Comment:
Couldn't figure out the variable needed to get the scroll bar to attach to the right edge of the screen, so this is a magic number for now